tests/test_pretest.py

```diff
@@ -405,6 +405,12 @@ def test_backend_acl_load(duthosts, enum_dut_hostname, tbinfo):
             pytest.fail("Backend acl not installed succesfully: {}".format(rule))
 
 
+# TODO may need to move this to health check
+def test_lldp_disabled_on_fanout(fanouthosts):
+    for fanout_name, fanouthost in fanouthosts.items():
+        assert fanouthost.is_lldp_disabled() is True
```

Comment:
I'm not clear about the background, why do we need to add it as a pretest testcase? Pretest would take effect on all platforms and topos, if fanouthost lldp is enabled, how many tests would be affected?

Reply:
That's right. I added this to test it. Where it should be invoked from is under discussion. The call could be moved to testbed health checker however the functions to check if LLDP is enabled will remain. I hoped to get comments on those for now.

Reply:
This test would only flag a failure if lldp is incorrectly disabled on fanout. It is hard to correlate this failure with other tests affected by bad lldp configuration on fanout.

It would be better to check and fix lldp configuration in other code, like testbed health monitoring and recovery code.

Can you remove this test? The enhancements to lib have value and should be kept.

Reply:
Thank you. I will close this PR.

Reply:
I have removed the test from pretest.